packages/message/src/Granules.ts

```diff
@@ -281,7 +279,7 @@ export const generateGranuleApiRecord = async ({
     files,
     error,
     published,
-    createdAt: createdAt || workflowStartTime,
+    createdAt,
```

Comment:
👍

Reply:
We went over this but could you explain the decision here re: workflowStartTime vs. createdAt.

Reply:
Sure -- there are philoshopical reasons for wanting this, but fundamentailly this method is taking a default and an option as arguments, when the logic/reason for that default being set it largely related to the calling method and as such it abstracts the branch logic without doing anything other than obscuring what's going on. 

In short - why pass `({..., value, valueDefault, ...})` only to have an assumbly method have `value || valueDefault`, when the decision to pass the value or not is entirely in the purview of the calling method and it obscures the underlying logic from maintainers working on the method?   It doesn't feel like an entirely useful abstraction, or at least not one that should be common to all callers of this method.